content/blog/2021/07/camel-k-release-1-5/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Camel K 1.5 release"
+date: 2021-07-13
+draft: false
+authors: [squakez]
+categories: ["Releases", "Camel K"]
+preview: "What's new in Camel K 1.5"
+---
+
+[Camel K 1.5 version](https://github.com/apache/camel-k/releases/tag/v1.5.0) is public since a few days. We're proud to announce this new release containing enhancements and fixes. We've worked hard to fix the most important issues discovered with an eye on stabilization and performances. This new release is based on **Camel Quarkus 2.0** and **Apache Camel 3.11**.
+
+Let's find out what's new in Camel K 1.5!
+
+## New Camel Quarkus and Quarkus versions
+
+**Camel Quarkus** runtime has become the building block on which the `Integration` is materialized. With the new 2.0 version out, there are great improvements which are leveraging the major release of **Quarkus 2.0**. We're inheriting all the good stuff in performance and components coverage that is introduced lately in this new version.
```

Comment:
Do we have any metrics to share on performance improvements and components coverage?

Reply:
I am not aware of any